hugegraph-test/src/main/java/com/baidu/hugegraph/core/VertexCoreTest.java

```diff
@@ -6667,6 +6669,37 @@ public void testQueryByPageWithInvalidPage() {
         });
     }
 
+    @Test
+    public void testQueryByPageWithSpecialBase64CharacterPage() {
+        // Assert decode '+' and '/' and '=' and space successfully
+        final String pageWith3Base64Chars = "AAAAADsyABwAEAqI546LS6WW57unBgA" +
+                                            "EAAAAAPB////+8H////4alhxAZS8va6" +
+                                            "opcAKpklipAAQAAAAAAAAAAQ==";
+
+        final String pageWithSpace = "AAAAADsyABwAEAqI546LS6WW57unBgAEAAAAAP" +
+                                     "B//// 8H////4alhxAZS8va6opcAKpklipAAQA" +
+                                     "AAAAAAAAAQ==";
+        Assert.assertNotNull(PageState.fromString(pageWith3Base64Chars));
+
+        byte[] decodePlus = PageState.fromString(pageWith3Base64Chars)
+                                     .position();
+        byte[] decodeSpace = PageState.fromString(pageWithSpace).position();
+
+        Assert.assertTrue(Arrays.equals(decodePlus, decodeSpace));
+        Assume.assumeTrue("Not support paging",
+                          storeFeatures().supportsQueryByPage());
+
+        HugeGraph graph = graph();
+        init100Books();
+
+        // Assert not throw exception when contains '+' or '/' or '=' or space
+        // Contains valid character '+' and '/' and '='
+        graph.traversal().V().has("~page", pageWith3Base64Chars)
+             .limit(10);
```

Comment:
assert empty?

Reply:
traversal is not empty, assert not null now